source/azure-mongodb-error-handling-demo/.gitignore

```diff
@@ -0,0 +1,33 @@
+HELP.md
+target/
+!.mvn/wrapper/maven-wrapper.jar
+!**/src/main/**/target/
+!**/src/test/**/target/
```

Comment:
probably only need the root `.gitignore`?

Reply:
probably but doesn't hurt.  :)